src/components/Editor.js

```diff
@@ -0,0 +1,56 @@
+export default function Editor({
+  $target,
+  initialState = { title: '', content: '' },
+  onEditing,
+}) {
+  const $div = document.createElement('div');
```

Comment:
```suggestion
  const $editor = document.createElement('div');
```
$div의 이름을 좀 더 명확하게하면 좋을 것 같아요!